src/Stripe.net/Services/Orders/OrderReturnOptions.cs

```diff
@@ -0,0 +1,14 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class OrderReturnOptions : BaseOptions
+    {
+        /// <summary>
+        /// List of items to return.
+        /// </summary>
+        [JsonProperty("items")]
+        public List<OrderReturnItemOptions> Items { get; set; }
```

Comment:
This naming feels like a new pattern. Usually we don't put the action/word name in the options class. But if we don't here, it conflicts with the one for Create which is the same hash but wit `currency` on top of it.

@ob-stripe Can you check and confirm that's the right naming scheme?

Reply:
Yeah, I think so. We're just "lucky" that in most cases the shape is shared between e.g. create and update methods, but more generally we probably shouldn't be sharing parameters, at least until the OpenAPI spec actually gives us that information.